src/v2/view-builder/views/IdentifierView.js

```diff
@@ -16,6 +17,33 @@ const Body = BaseForm.extend({
     return loc('oform.next', 'login');
   },
 
+  saveForm() {
+    // Ideally this can be added to a "preSaveForm" handler - but keeping this here for now.
+    if (!this.settings.get('features.deviceFingerprinting')) {
```

Comment:
`features.deviceFingerprinting` has been checked twice: one here and another in the interceptor config.
Similarly for checking `/idp/idx/identify` (we modify IdentifyView hence we already know figure-print coming from identify step but we check again at interceptor.)
I'd try to think if there is way to eliminate such redundancy. 
**maybe** in interceptor, we only need to check the availability of `deviceFingerprint` ?

Reply:
Great suggestion - we should make sure the `IdentifierView` keeps this decision logic, so I updated this PR to remove the duplicate checks.